### DISCUSSION TITLE
物件交互界面控制

### DISCUSSION THREAD

**Discussion by 0kk470** [jynew/jynew#1303] [Ideas]
**功能描述**
提供对话、使用物品等操作的入口。

**UI路径导航**
无需导航，直接两个按键映射操作。
![image](https://user-images.githubusercontent.com/25216715/199403687-fa868ebd-b82f-4e66-b6ad-3b68d1a4032e.png)

**Comment by MephistoSatanDevil** [comment]
这个界面其实目前最大问题不在于操作，而在于想要准确的触发到对应物品的交互界面比较困难，而且游戏中经常出现两个人站得比较近或者离宝箱比较近的时候，玩家想要从一个交互对象转到另一个交互对象得来回走好几次（比如金群MOD中南贤和旁边的柜子）。

**Reply by 0kk470** [reply]
> 这个界面其实目前最大问题不在于操作，而在于想要准确的触发到对应物品的交互界面比较困难，而且游戏中经常出现两个人站得比较近或者离宝箱比较近的时候，玩家想要从一个交互对象转到另一个交互对象得来回走好几次（比如金群MOD中南贤和旁边的柜子）。

这个还好吧，代码里还做了方向判定的，后面再优化下碰撞判定
